docs/pages/access-controls/access-request-plugins/ssh-approval-jira.mdx

```diff
@@ -7,8 +7,6 @@ This guide will talk through how to set up Teleport with Jira. Teleport's Jira
 integration allows you to treat Teleport access and permission requests using
 Jira tickets.
 
-<Details title="This integration is hosted on Teleport Cloud" open={false}>
-
 (!docs/pages/includes/plugins/enroll.mdx name="the Jira integration"!)
 
 </Details>
```

Comment:
This is almost certainly incorrect. You know have a closing `</Details>` tag without an opening one.